.trivyignore

```diff
@@ -0,0 +1,6 @@
+# Patched. https://ubuntu.com/security/CVE-2024-24788
```

Comment:
file duplicated in `golang-rock/1.22-24.04/.trivyignore`. this was not the case in #4